### DISCUSSION TITLE
New Wifi MOD uses RTS5227S (use pcie x1) chipset for SD card which can be desoldered/replaced with a BCM94360CS2

### DISCUSSION THREAD

**Discussion by piotrasd** [osy86/HaC-Mini#560] [General]
that not issue but is interesting and maybe someone will want dig bit more in topic

In NUC8i7BEH chipset RTS5227S (use pcie x1) for SD card can be desoldered/replaced with a BCM94360CS2(or BCM942602CS) to get native AirDrop/Handoff support.

I didn't check yet but possible same chip we have in Skull Hades 

Look interesting to keep second M.2 slot for something else

more details here
https://www.chiphell.com/thread-2004976-1-1.html
https://www.mydigit.cn/forum.php?mod=viewthread&tid=34830&extra=&page=1

Source
https://github.com/randyzhong/OS-X-NUC8i7BEH-Clover

**Comment by osy** [comment]
Interesting but I think for us, the Dell card is good enough if you need both m2 ports and still want handoff.

**Comment by piotrasd** [comment]
somebody can confirm if sidecar working with dell card ?

**Comment by lmshsqlc** [comment]
> somebody can confirm if sidecar working with dell card ?

It is confirmed working. However sometimes people just reported not working including me.  it somehow just came working for some reason.

**Comment by osy** [comment]
I think the people who reported it not working don't have iCloud set up properly.

**Comment by KrisCris** [comment]
> Interesting but I think for us, the Dell card is good enough if you need both m2 ports and still want handoff.

Actually I found another way using native apple card like bcm943602cs while retain the 2 m.2 slot. However, only SSDs without chip on the back are able to fit. But its better than nothing.

![image](https://user-images.githubusercontent.com/38860226/72548222-3bbc4e80-38c9-11ea-9468-5de8fc25623c.png)
![IMG_20200116_181128](https://user-images.githubusercontent.com/38860226/72548352-732afb00-38c9-11ea-8d2f-01b862d1615c.jpg)

**Comment by chrisallen** [comment]
Thanks, @KrisCris! I like that option. I'm going to order one and give it ago!

**Comment by crhendo** [comment]
@KrisCris that looks very interesting. I have been trying to locate it using Google search without any luck. Do you have a link you could share?

**Comment by KrisCris** [comment]
> @KrisCris that looks very interesting. I have been trying to locate it using Google search without any luck. Do you have a link you could share?

I bought that from Taobao (the china mainland version of AliExpress), not sure if its available for you.
https://item.taobao.com/item.htm?spm=a1z09.2.0.0.42d32e8dhL9hKX&id=606077440886&_u=k2d29doj502c

Edit:
I think we could talk about that adapter under #86 , since this is unrelated to the current issue.
